data/moves.js

```diff
@@ -10518,17 +10501,16 @@ let BattleMovedex = {
 		effect: {
 			duration: 1,
 			noCopy: true,
-			onStart: function (target, source, source2, move) {
+			onStart: function () {
```

Comment:
Does TS not allow empty args anymore? We should probably allow them for consistency.